weblate/utils/search.py

```diff
@@ -282,6 +286,8 @@ def query_sql(obj):
             return has_sql(obj.text)
         if obj.fieldname == "is":
             return is_sql(obj.text)
+        if obj.text.startswith("="):
```

Comment:
Looking into text is not the best approach (`source:"=value"` should probably search for the `=value` instead of doing exact match).

But it's probably good enough until the query parser is rewritten (see https://github.com/WeblateOrg/weblate/issues/3229).

Reply:
I was also honestly wondering the same while implementing this. That it should probably search for `=value`. But since the parser always results in it being a Term instance, I couldn't figure a better way of segregating it apart from looking into the text. Let me know if you think there is a different better way I can do this.